src/Angor/Client/Pages/Signatures.razor

```diff
@@ -376,6 +380,60 @@
         }
     }
 
+    private async Task ApproveAllSignatures()
```

Comment:
@dangershony is there a better way to do this?
i added another function because approvesignature() resets the password each time, and i didn't want to add a boolean in that function.